Modelica/Electrical/Analog/Examples/Lines/CompareCmatrix.mo

```diff
@@ -0,0 +1,273 @@
+within Modelica.Electrical.Analog.Examples.Lines;
+model CompareCmatrix
+  extends Modelica.Icons.Example;
+
+//********************************************
+model MTL_Cmatrix
+  "Y-matrix with capacitive coupling for a multi-conductor line"
+  parameter Integer n(final min=1) = 3 "Number of conductors";
+  parameter Real C[n,n]( each final unit="F/m")  "Capacitance matrix; off-diagonal elements are negative";
```

Comment:
It is common for lines to use parameters per unit length.
This is unavoidable, since actual inductances and capacitanses of lines are proportional to line lengths.
The units proposed here are SI, even though do not appear in Modelica.Units.SI.

Reply:
The implicit question is whether where is a type in Modelica.Units.SI that could be used. Permittivity has the same unit (F/m) - but I don't understand whether it is the same.

Reply:
oops!
A few year ago, after my first PRs related to lines, new units have been added. 
I forgot this here.
I've just updated the PR using ResistancePerUnitLength, InductancePerUnitLength, CapacitancePerUnitLength..